server/app.js

```diff
@@ -1,16 +1,22 @@
 const express = require('express')
 const app = express()
 
-const { getTopics, getApi } = require('../server/controllers/topics.controllers')
+const { getTopics,
+         getApi,
+         getArticleByID } = require('../server/controllers/topics.controllers')
 
-const { handle404s } = require('./errors')
+const { handle404s, handle400s, handle500s } = require('./errors')
 
 app.use(express.json())
```

Comment:
Why is this line of code here when we aren't writing anything involving bodies yet?